platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh

```diff
@@ -37,7 +37,6 @@ remove_python_api_package() {
 if [[ "$1" == "init" ]]; then
     modprobe i2c-dev
     modprobe i2c-mux-pca954x force_deselect_on_exit=1
-    modprobe dell_ich
```

Comment:
As per https://github.com/Azure/sonic-buildimage/pull/7336, should you revert the changes to this file?

Reply:
No harm in removing this change in 201811. Will commit the new changes.